applications/nrf_desktop/README.rst

```diff
@@ -205,7 +205,15 @@ Every of these reports uses predefined report format and provides the given info
 For example, the mouse motion is forwarded as HID mouse report.
 
 An nRF Desktop device supports the selected subset of the HID input reports.
-For example, the nRF Desktop keyboard reference desing (nrf52kbd_nrf52832) supports HID keyboard report, HID consumer control report and HID system control report.
+For example, the nRF Desktop keyboard reference design (nrf52kbd_nrf52832) supports HID keyboard report, HID consumer control report and HID system control report.
+
+The following HID protocols are supported by nRF Desktop for HID input reports:
```

Comment:
The HID protocol refers to HID reports in general, not only to HID input reports (e.g. we have HID boot keyboard LEDs report that is an HID output report)

Reply:
yes, but HID input reports can use only either boot or report protocol.

Reply:
I would just suggest to separate HID report types (input, output and feature) from HID protocols (boot, report). I think that could be more understandable for the reader

Reply:
Also under this section you currently mention data exchanged by device and host (and section refers only to HID input reports that come from device to the host)